Add config flow to Random (#29528)
home-assistant/home-assistant.github.io: source/_integrations/random.markdown

The `random` integration simply creates random values or state. This can be useful if you want to test automation rules or run an interactive demo. It generates a new state every time it is polled.

## Configuration
The preferred way to configure random helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=page.ha_domain title=page.title %} option.

> To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `random:` to your `configuration.yaml` first, then you can use the UI.

Reviewer: ```suggestion
To be able to add Helpers via the user interface, you should have `default_config:` in your `configuration.yaml`. It should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `random:` to your `configuration.yaml` first, then you can use the UI.
```

The `random` integration simply creates random values or state. This can be useful if you want to test automation rules or run an interactive demo. It generates a new state every time it is polled.

## Configuration

> The preferred way to configure timer helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=random title="Random" %} option.

Reviewer: ```suggestion
The preferred way to configure random helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=page.ha_domain title=page.title %} option.
```

> You can also select the following button to be redirected to the Helpers page of your Home Assistant instance.

Reviewer: ```suggestion
```

{% my helpers badge %}

> To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `timer:` to your `configuration.yaml` first, then you can use the UI.

Reviewer: ```suggestion
To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `random:` to your `configuration.yaml` first, then you can use the UI.
```

The `random` integration simply creates random values or state. This can be useful if you want to test automation rules or run an interactive demo. It generates a new state every time it is polled.

The random binary sensor is creating random states (`true`, 1, `on` or `false`, 0, `off`).

> The random sensor is creating random sensor values (integers) out of a given range. Returned values form a [discrete uniform distribution](https://en.wikipedia.org/wiki/Discrete_uniform_distribution), meaning that each integer value in the range configured is equally likely to be drawn.

Reviewer 2: ```suggestion
The random sensor creates random sensor values (integers) out of a given range. Returned values form a [discrete uniform distribution](https://en.wikipedia.org/wiki/Discrete_uniform_distribution), meaning that each integer value in the range configured is equally likely to be drawn.
```

## Configuration

> The preferred way to configure timer helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and click the add button; next choose the {% my config_flow_start domain=random title="Random" %} option.

Reviewer 2: ```suggestion
The preferred way to configure timer helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=random title="Random" %} option.
```

> You can also click the following button to be redirected to the Helpers page of your Home Assistant instance.

Reviewer 2: ```suggestion
You can also select the following button to be redirected to the Helpers page of your Home Assistant instance.
```

{% my helpers badge %}

To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `timer:` to your `configuration.yaml` first, then you can use the UI.

## YAML configuration

> ### Binary Sensor

Reviewer 2: ```suggestion
### Binary sensor
```
